jsign-crypto/src/main/resources/META-INF/services/net.jsign.ks.JsignKeyStore

```diff
@@ -0,0 +1,21 @@
+net.jsign.ks.AwsKeyStore

```

Comment:
The service file can be automatically generated by annotating the implementations with @MetaInfServices(JsignKeyStore.class)

Reply:
Fixed